js/cm-api-v2/GetTaxonomyGroup.js

```diff
@@ -0,0 +1,22 @@
+// DocSection: cm_api_v2_get_taxonomy_group
+// Tip: Find more about JS/TS SDKs at https://docs.kontent.ai/javascript
+// Using ES6 syntax
+import { ManagementClient } from '@kentico/kontent-management';
+
+const client = new ManagementClient({
+  projectId: '<YOUR_PROJECT_ID>',
+  apiKey: '<YOUR_API_KEY>'
+});
+
+client.getTaxonomy()
```

Comment:
The sample values for identifiers (id, codename, external_id) should match those in the API docs. See the Taxonomy object at https://docs.kontent.ai/reference/management-api-v2#section/Taxonomy-group-object.